implementations/rust/ockam/ockam_api/src/cli_state/identities.rs

```diff
@@ -55,6 +55,18 @@ impl CliState {
             .await
     }
 
+    #[instrument(skip_all, fields(name = %name, identifier = %identifier, vault_name = %vault_name))]
+    pub async fn create_identity_with_identifier(
```

Comment:
What is the difference between this one and `store_named_identity`? Is it that we instantiate the Vault here? What is the idea behind that?

Reply:
I didn't see that method, I'll remove `create_identity_with_identifier`.